### PR TITLE
Add workspaces template

### PR DESCRIPTION
**Pull request by gdavidkov**

### CODE REVIEW

File: io-connect-desktop-workspaces/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Glue42 Workspaces</title>
```
Comment (arjunah) [inline io-connect-desktop-workspaces/public/index.html:27]: io.Connect Workspaces

File: io-connect-desktop-workspaces/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "Glue42 Workspaces",
```
Comment (arjunah) [inline io-connect-desktop-workspaces/public/manifest.json:2]: io.Connect Workspaces App

File: io-connect-desktop-workspaces/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "Glue42 Workspaces",
```
Comment (arjunah) [inline io-connect-desktop-workspaces/public/manifest.json:3]: io.Connect Workspaces App

**Review by arjunah — CHANGES_REQUESTED** [review]

**Review by arjunah — APPROVED** [review]

**Review by SvetozarMateev — APPROVED** [review]

File: io-connect-desktop-workspaces/README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Workspaces in React
+This project provides a starter for React based Workspaces applications in io.Connect Desktop and io.Connect Browser.
```
Comment (arjunah) [inline io-connect-desktop-workspaces/README.md:2]: Template for creating a custom Workspaces App \[for \*\*io.Connect Desktop**](https://docs.interop.io/desktop/capabilities/windows/workspaces/overview/index.html#extending_workspaces) or \[for \*\*io.Connect Browser**](https://docs.interop.io/browser/capabilities/windows/workspaces/workspaces-app/index.html) projects.

\## Prerequisites

For \*\*io.Connect Desktop** projects, \*\*io.Connect Desktop** 9.0 or newer is required. For \*\*io.Connect Browser** 
 projects, \*\*io.Connect Browser** 3.0 or newer is required.

\## Usage

\- Run \`npm install\` to install all dependencies.
\- Run \`npm run start\` to start the app.
\- For \*\*io.Connect Desktop** projects, copy the \`workspaces.json\` configuration file and replace the existing Workspaces App definition in the \`%LocalAppData%\interop.io\io.Connect Desktop\Desktop\config\apps\` folder. For \*\*io.Connect Browser** projects, see the \[Enabling Workspaces](https://docs.interop.io/browser/capabilities/windows/workspaces/enabling-workspaces/index.html) documentation.
\- Start \*\*io.Connect Desktop** to start using and modifying your Workspaces App.

**Review by arjunah — CHANGES_REQUESTED** [review]

**Review by arjunah — APPROVED** [review]

**Pull request merged**